core/src/test/resources/spark-events-profiling/driverlog

```diff
@@ -0,0 +1,3907 @@
+23/11/07 10:59:24.335 main INFO DriverLogger: Added a local log appender at: /tmp/spark-33ef7368-7a1e-44a1-ab42-ccbd843dabf4/__driver_logs__/driver.log
```

Comment:
how big is this file, do we really want to check it in?  can we generate it on a fly or specific parts of it on the fly for unit tests and do soethig more in integration tests

Reply:
Filesize is 676K(which can be reduced further). I checked in the file for this test because generating GPU driver log  would require spark-rapids jar. Please let me know if it's okay to keep checkin the file instead of generating on the fly for this case.

Reply:
676K is pretty large, can we truncate it to be much smaller.  do we have an issue to do more integration testing?

Reply:
Reduced the file size to 52K. Filed a follow on issue for adding integration tests - https://github.com/NVIDIA/spark-rapids-tools/issues/658